digital_land/phase/lookup.py

```
@@ -189,14 +214,85 @@ def process(self, stream):
 class EntityLookupPhase(LookupPhase):
     entity_field = "entity"
 
+    def __init__(
+        self,
+        lookups={},
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)